From 8631e84f1f296a5c98dbebe119211b9f4fc1edca Mon Sep 17 00:00:00 2001
From: testuser <testuser@gmail.com>
Date: Tue, 16 Apr 2019 11:23:01 +0000
Subject: [PATCH] patch test

---
 fileone | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/fileone b/fileone
index a732d9d..58d2b18 100644
--- a/fileone
+++ b/fileone
@@ -3,3 +3,9 @@ line 2 file 1
 changes made by test1 user
 test user added a line
 difference added modified file
+
+patch testing lines 
+hello 
+
+how are you 
+testing file 
-- 
1.8.3.1

